src/components/external/DisputeManagement/components/DefendDisputeFlow/DefendDisputeFlow.scss

```
@@ -107,6 +107,19 @@
         color: style.token(color-label-secondary);
         font-size: style.token(text-caption-font-size);
     }
+
+    &__dropdown-list {
+        max-height: 270px;
+        overflow: scroll;
+
+        &--mobile {
+            left: 0;
+            position: fixed;
+            top: 0;
+            width: 80%;
+            z-index: 30;
+        }
+    }
```

Instead of duplicating this, what do you think about making this a mixin?

```scss
@mixin adyen-pe-dropdown-overrides {
    &__dropdown-list {
        max-height: 270px;
        overflow: scroll;

        &--mobile {
            left: 0;
            position: fixed;
            top: 0;
            width: 80%;
            z-index: 30;
        }
    }
}
```

```suggestion
    @include adyen-pe-dropdown-overrides;
```